### DISCUSSION TITLE
I would like to ask if the GitHub Copilot feature is included in the GitHub Enterprise version or not. If so, how can we activate this feature?

### DISCUSSION THREAD

**Discussion by ngocduysdtq** [community/community#49942] [Enterprise] [answered]
I am a customer of GitHub Enterprise and I am currently looking into the GitHub Copilot feature to use in our company's development projects. However, I haven't found specific information on whether this feature is included in the GitHub Enterprise version or not.

Therefore, I would like to ask if the GitHub Copilot feature is included in the GitHub Enterprise version or not. If so, how can we activate this feature?

**Comment by neutrinomuon** [comment]
Thanks for pointing it out :)

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @ngocduysdtq 👋🏼 , 

Thanks for asking - GitHub Copilot is available to GitHub Enterprise users with Copilot for Business at an additional cost. Check out our [documentation](https://docs.github.com/en/enterprise-cloud@latest/copilot/overview-of-github-copilot/about-github-copilot-for-business) for pricing and how to enable it. 

Let us know if you have any further questions.
